### PR TITLE
Generate featured contents dyanamically

### PR DESCRIPTION
**Pull request by lenkon**
### Project name: Capstone 1 - Concert Page

Project description: This is a capstone project for the first module where I create a concert festival page using HTML, CSS and JavaScript.

### Project requirements

- Personalize the content of the page. Choose a topic that is different than the one in the original design.
- Follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
  - Colors.
  - Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
  - Layout: composition and space between elements.
- The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.
- Use a CSS framework (for example Bootstrap) for styling, if you want to.
- Build only these 2 pages:
  - The *home page*.
  - The *about page*.
- Each of these pages should have versions for 2 different screen sizes: 
  - Mobile: up to 768px wide.
  - Desktop: 768px or wider.

- Interactions
  - Links
    - The *home page* should have a link in the menu to the *about page*.
    - The logo in the header links to the *home page*.
  - Mobile menu
    - When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
    - There are no guidelines for the mobile menu in the docs, but implement it so it is consistent with the design (colors, typography, spacings, etc.).
    - The mobile menu should have a close (X) button that closes the menu.
- Dynamic page
  - The section "Featured speakers" should be created dynamically in JavaScript.
  - Use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

Project requirements link: https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md

### CODE REVIEW

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @lenkon , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Descriptive pull request message ✔️
- No linter error ✔️
- Amazing design ✔️
- Well documented project ✔️

there

## Required Changes ♻️
- [x] Let's add in readme a video containing a highlight about a piece of the code that you find interesting, or you are proud of making

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lenkon** [comment]
Hi @uwadonat, Thanks for the valuable feedback. I have created the project presentation video and have added the link to the README file.

File: README.md [hunk 1]
```diff
@@ -63,7 +63,8 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://github.com/lenkon/Portfolio)
+- [Live Demo Link](https://lenkon.github.io/)
+- [Project Presentation Video Link](https://www.loom.com/share/dd0b042569e94d11ac0cb24b3e31e58e)
```
Comment (fobadara) [inline README.md]: [REQUIRED] Your video length should be a maximum of 5 minutes. You didn't need to present the entire code. All you should have presented in your code is the one you found interesting then you move on to the output in the browser.
[REQUIRED] You also forgot to highlight a piece of code you found interesting or proud of making.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @lenkon, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing
✅ Project looks professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- creating a link that returns the user to the homepage instead of the about page. The previous reviewer should have marked this has a required change.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (lenkon) [reply README.md]: Hi @fobadara, Thanks for the feedback. I have a doubt. So do I have to highlight only one piece of code or more than one? Can the video length be just about 2 minutes also?

Comment (lenkon) [reply README.md]: Hi @fobadara, I have created a new video that is less than 5 minutes and I have also highlighted a couple of code blocks that I found interesting.

**Review by sumairq — APPROVED** [review]
### Hi @lenkon , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**